vllm/v1/engine/core_client.py

```diff
@@ -453,6 +459,9 @@ def _init_core_engines(
     def shutdown(self):
         self._finalizer()
 
+    def get_gpu_blocks(self) -> VllmConfig:
+        return self.vllm_config
```

Comment:
We should be able to remove this

Reply:
Because `num_gpu_blocks` this value is in _wait_for_engine_startup update, so we need from get new vllm_config, if we use vllm_config  in `LoggingStatLogger.__init__()` , it `num_gpu_blocks` values is `None.

Reply:
Hmm, you're saying a new instance of `VllmConfig` gets created? Where does that happen?

```
self.vllm_config.cache_config.num_gpu_blocks = message_dict[
                'num_gpu_blocks']
```

This just modifies an existing instance  - is it not the same instance we can pass to `LoggingStatLogger.__init__()` ?

Reply:
ok, I understand, because it is passed by pointer